react/week1/todolist/src/TodoList.js

```diff
@@ -0,0 +1,10 @@
+import React from 'react'
+import Todo from './Todo'
+export default function TodoList({todos}){
+    return(
+       todos.map(todo=> {
+           return <Todo key={todo.description} todo ={todo}/>
+       })
```

Comment:
In this case when you want to do JavaScript inside of the HTML you need to wrap the JavaScript in {curly brackets} and here enclose it in the Fragment <></>

Reply:
I had wrapped it in curly brackets,is not that right? todo ={todo} ? yes,understood about fragment.